applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -319,6 +304,62 @@ config BT_AUDIO_BROADCAST_24_2_2
 	  24kHz, 48kbps, 4 retransmits, 60ms transport latency, and 40ms presentation delay
 endchoice
 
+config BT_AUDIO_BROADCAST_NAME
+	string "Broadcast name"
+	depends on TRANSPORT_BIS
+	default "NRF5340_BROADCASTER"
+	help
+	  Name of the broadcast, not the same as BT_DEVICE_NAME
+
+config BT_AUDIO_BROADCAST_NAME_ALT
+	string "Alternative broadcast name"
+	depends on TRANSPORT_BIS
+	default "NRF5340_BROADCASTER_ALT"
+	help
+	  Alternative name of the broadcast
+
+config BT_USE_AUDIO_BROADCAST_NAME_ALT
+	bool "Use the alternative Broadcast name"
+	default n
+	depends on TRANSPORT_BIS
+	help
+	  Use the alternative Broadcast name.
```

Comment:
Any reason Broadcast is sometimes capitalized, sometimes isn't?

Reply:
Not a good reason, will change it.